app/common/elasticsearchclient.py

```diff
@@ -493,110 +493,63 @@ def get_label_for_eco_code(self, code):
             return hit['_source']
 
     def get_evidence(self,
-                     targets=[],
-                     diseases=[],
-                     evidence_types=[],
-                     datasources=[],
-                     datatypes=[],
-                     gene_operator='OR',
-                     object_operator='OR',
-                     evidence_type_operator='OR',
                      **kwargs):
         params = SearchParams(**kwargs)
         if params.datastructure == SourceDataStructureOptions.DEFAULT:
             params.datastructure = SourceDataStructureOptions.FULL
-        '''convert boolean to elasticsearch syntax'''
-        gene_operator = getattr(BooleanFilterOperator, gene_operator.upper())
-        object_operator = getattr(BooleanFilterOperator, object_operator.upper())
-        evidence_type_operator = getattr(BooleanFilterOperator, evidence_type_operator.upper())
-        '''create multiple condition boolean query'''
-        conditions = []
-        if targets:
-            conditions.append(self.get_complex_target_filter(targets, gene_operator))
-        if diseases:
-            conditions.append(self.get_complex_disease_filter(diseases, object_operator, is_direct=False))#temporary until params.is_direct is supported again
-        if evidence_types:
-            conditions.append(self._get_complex_evidence_type_filter(evidence_types, evidence_type_operator))
-        if datasources or datatypes:
-            requested_datasources = []
-            if datasources:
-                requested_datasources.extend(datasources)
-            if datatypes:
-                for datatype in datatypes:
-                    requested_datasources.extend(self.datatypes.get_datasources(datatype))
-            requested_datasources = list(set(requested_datasources))
-            conditions.append(
-                self._get_complex_datasource_filter_evidencestring(requested_datasources, BooleanFilterOperator.OR))
-        if params.pathway:
-            pathway_filter = self._get_complex_pathway_filter(params.pathway)
-            if pathway_filter:
-                conditions.append(pathway_filter)
-        if params.uniprot_kw:
```

Comment:
It would be good to have them here as well for consistency. these kind of queries can be useful even at the evicende level in the API if if at the moment they are not used in the OT Platform web app.